lib/toc_obj.js

```diff
@@ -8,28 +8,26 @@ const parseHtml = html => {
   return handler.dom;
 };
 
-const getId = ele => {
-  const { id } = ele.attribs;
-  const { parent } = ele;
-  return id || (!parent ? null : getId(parent));
+const getId = ({ attribs, parent }) => {
+  return attribs.id || (!parent ? null : getId(parent));
 };
 
 function tocObj(str, options = {}) {
-  options = Object.assign({
+  const { min_depth, max_depth } = Object.assign({
     min_depth: 1,
     max_depth: 6
   }, options);
 
-  const headingsSelector = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'].slice(options.min_depth - 1, options.max_depth).join(',');
+  const headingsSelector = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'].slice(min_depth - 1, max_depth);
```

Comment:
Array has `includes()` as well, remove unnecessary `.join()`.

Reply:
Array also works better since the value must be exact match.
Currently headingsSelector also matches `h` tag.